### PR TITLE
radio group to select one among different data sources

### PR DESCRIPTION
**Pull request by championswimmer**
solves this issue https://github.com/fossasia/open-event-webapp/issues/305

Signed-off-by: Arnav Gupta <championswimmer@gmail.com>

### CODE REVIEW

**Comment by championswimmer** [comment]
screenshot from 2016-06-19 01-22-21
screenshot from 2016-06-19 01-22-26
screenshot from 2016-06-19 01-22-31

File: src/generator/www/js/form.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      function() {
+          if ($(this).is(':checked')) {
+
+            if ($(this).val() == 'mockjson') {
+              console.log('jsonupload')
+              $('#jsonupload-input').hide(100);
+              $('#eventapi-input').hide(100);
+            }
+
+            if ($(this).val() == 'jsonupload') {
```
Comment (aayusharora) [inline src/generator/www/js/form.js]: @championswimmer , correct it please .

**Comment by aayusharora** [comment]
@championswimmer , what is the need of the field default Mock API ?

**Comment by championswimmer** [comment]
We don't need to upload anything then.
Using this option means, it'll take files from mockjson folder.
Fastest way to test

**Comment by aayusharora** [comment]
Looks good to me @championswimmer

**Pull request merged**